charts/portal/values.yaml

```diff
@@ -128,6 +128,8 @@ frontend:
       name: "docker.io/tractusx/portal-frontend"
       portaltag: 37cd674aa1324f0fbd9c7662b5e43fde89e1a748
       pullPolicy: "IfNotPresent"
+      # -- Pull secrets for private docker registry
+      imagePullSecrets: []
```

Comment:
I think it would be more consistent to call the value `pullSecrets` overall as the path already includes `image` e.g. `image.imagePullSecrets`